webapp/app.py

```diff
@@ -73,11 +73,28 @@ def thank_you():
 
 @open_id.after_login
 def after_login(resp):
-    flask.session["macaroon_discharge"] = resp.extensions["macaroon"].discharge
-    flask.session["openid"] = {
-        "identity_url": resp.identity_url,
-        "email": resp.email,
+    root = flask.session["macaroon_root"]
+    discharge = resp.extensions["macaroon"].discharge
+    authorization_header = get_authorization_header(root, discharge)
```

Comment:
Do you really need to send this? I don't think the API docs suggest it's needed, and it would be odd since the exact same information is sent in the `data`.

Reply:
If you didn't confirm that you need this, please try without. And if you don't need it, you can delete the `get_authorization_header` function.

Reply:
Yeah, I just tested it out and it seems to be enough to just POST the payload